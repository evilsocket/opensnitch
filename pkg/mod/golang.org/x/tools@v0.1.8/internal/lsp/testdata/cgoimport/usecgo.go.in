package cgoimport

import (
	"golang.org/x/tools/internal/lsp/cgo"
)

func _() {
	cgo.Example() //@godef("ample", funccgoexample),complete("ample", funccgoexample)
}
