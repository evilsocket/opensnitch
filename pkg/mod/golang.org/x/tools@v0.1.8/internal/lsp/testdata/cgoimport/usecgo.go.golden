-- funccgoexample-definition --
cgo/declarecgo.go:18:6-13: defined here as ```go
func cgo.Example()
```

[`cgo.Example` on pkg.go.dev](https://pkg.go.dev/golang.org/x/tools/internal/lsp/cgo?utm_source=gopls#Example)
-- funccgoexample-definition-json --
{
	"span": {
		"uri": "file://cgo/declarecgo.go",
		"start": {
			"line": 18,
			"column": 6,
			"offset": 151
		},
		"end": {
			"line": 18,
			"column": 13,
			"offset": 158
		}
	},
	"description": "```go\nfunc cgo.Example()\n```\n\n[`cgo.Example` on pkg.go.dev](https://pkg.go.dev/golang.org/x/tools/internal/lsp/cgo?utm_source=gopls#Example)"
}

-- funccgoexample-hoverdef --
```go
func cgo.Example()
```

[`cgo.Example` on pkg.go.dev](https://pkg.go.dev/golang.org/x/tools/internal/lsp/cgo?utm_source=gopls#Example)
